docs/dev/plugins/cloudformation/deployment/deployment-workflow.md

```diff
@@ -0,0 +1,45 @@
+# Deployment Workflow
+
+As explained in the [transformation workflow](../transformation/transformation-workflow.md) chapter, the target artifact generated by the transformation is not a single CloudFormation template, but rather a template, additional files and scripts all needed for the deployment.
+
+Therefore, additional steps are required besides just simply deploying the template to CloudFormation. This chapter explains these steps as the intended workflow of the deployment.
+
+## Deployment Step-by-Step
+
+Before the deployment begins, we start with the target artifact generated by the transformation with the CloudFormation plugin.
+
+![Target Artifact](img/deployment-step-0.jpg)
+
+>**Note**: The images used in this document are taken from the deployment of the LAMP example explained in [Transformation by Example](../transformation/transformation-examples.md) chapter. Other transformations may differ in specifics but the general deployment behaviour is the same.
+
+The deployment can be divided into several steps. These steps are done through the use of bash scripts. For further information about these scripts, please refer to the [target artifact](target-artifact.md) chapter. The execution of these steps follows the order specified below.
+
+### 1. Create an S3 Bucket
+
+![Deployment Step 1](img/deployment-step-1.jpg)
```

Comment:
I think using UML state diagrams in all images would have been a better fit (yeah I know, too late).